broker/broker.go

```diff
@@ -385,10 +391,21 @@ func (b *Broker) LastOperation(ctx context.Context, instanceID string, pollDetai
 		}
 	}
 
+	if operation.TimeOut != "" {
+		timeOutParsed, err := time.Parse(time.RFC3339, operation.TimeOut)
+		if err != nil {
+			return brokerapi.LastOperation{}, errors.Wrap(err, "Failed to parse time out string")
+		}
+
+		if time.Now().Unix() > timeOutParsed.Unix() {
```

Comment:
[The `After` method](https://pkg.go.dev/time#After) is probably a more idiomatic way of expressing this

```
if time.Now().After(timeOutParsed) {

}
```

Reply:
Update and also changed in the test case.